ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.20__multi-nightly.yaml

```diff
@@ -2332,6 +2332,25 @@ tests:
       WORKFLOW_TYPE: cucushift
       YQ_VERSION: 4.30.5
     workflow: openshift-e2e-ibmcloud-heterogeneous-test-qe
+- as: azure-short-cert-rotation-f11
```

Comment:
If you decided to use multi payload to run installation on arm arch nodes, you do not need to add it again in openshift-openshift-tests-private-release-4.19__arm64-nightly.yaml.

For the multi-nighlty ci, either arm or amd arch nodes can be used to run the install, once that, the ones in arm64-nightly or amd64-nightly can be saved.

Reply:
Here is an [example](https://github.com/openshift/release/blob/f30af6eb965e8ebbe1012be287c0d8686dbaf9d7/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.20__multi-nightly.yaml#L984-L998), when using arm arch nodes, need to set `COMPUTE_NODE_TYPE` and `OCP_ARCH`

Reply:
Yes I will remove those when  multi job got success